New API doc changes (#2935)
timescale/docs: api/add_compression_policy.md

|`initial_start`|TIMESTAMPTZ|Time the policy is first run. Defaults to NULL. If omitted, then the schedule interval is the interval from the finish time of the last execution to the next start. If provided, it serves as the origin with respect to which the next_start is calculated |
|`timezone`|TEXT|A valid time zone. If `initial_start` is also specified, subsequent executions of the compression policy are aligned on its initial start. However, daylight savings time (DST) changes may shift this alignment. Set to a valid time zone if this is an issue you want to mitigate. If omitted, UTC bucketing is performed. Defaults to `NULL`.|
|`if_not_exists`|BOOLEAN|Setting to `true` causes the command to fail with a warning instead of an error if a compression policy already exists on the hypertable. Defaults to false.|

> |`compress_created_before`|INTERVAL|Specification of cut-off point where any chunks with creation time older than this timestamp should be compressed. The cut-off point is computed as `now() - compress_created_before`. Defaults to `NULL`. Not supported for continuous aggregates yet.|

Reviewer: ```suggestion
|`compress_created_before`|INTERVAL|Chunks with creation time older than this are compressed. The cut-off point is computed as `now() - compress_created_before`. Defaults to `NULL`. Not supported for continuous aggregates.|
```

Author: Done.

SELECT add_compression_policy('cpu', INTERVAL '60d');
```

Add a policy to compress chunks created 3 months before on the 'cpu' hypertable.

``` sql
SELECT add_compression_policy('cpu', compress_created_before => INTERVAL '3 months');
```

> 

Reviewer: I think you need to elaborate on this a little. For example, if I have a chunk interval of 1 month and specify a `compress_created_before` of three months, what is the most recent data that will be compressed? It can be computed using the information here, but it would be good to give some more context to make the text a little easier to work with.

Author: @mkindahl there's no connection between chunk interval and chunk creation time.

Reviewer: Yes, but it would be better with some more context just to make it easier to understand. (I was thinking about the situation where the current data is inserted and hence the creation time will be the same as the insertion time of the first row.)
